packages/flatpak.rb

```diff
@@ -0,0 +1,90 @@
+require 'package'
+
+class Flatpak < Package
+  description 'Flatpak is a system for building, distributing, and running sandboxed desktop applications on Linux.'
+  homepage 'https://flatpak.org'
+  @_ver = '1.10.1'
+  version @_ver
+  compatibility 'all'
+  source_url "https://github.com/flatpak/flatpak/releases/download/#{@_ver}/flatpak-#{@_ver}.tar.xz"
+  source_sha256 'c1354f42bf3b5d51aeb4028c9b62fd4ffc673ef2ff6e583c17777f5dafdbdcb7'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/flatpak-1.10.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/flatpak-1.10.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/flatpak-1.10.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/flatpak-1.10.1-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: 'ee4cf9fe437d1aa7b6804d0b323c81c9df25cd14a6124bc5d624ad3238df6875',
+     armv7l: 'ee4cf9fe437d1aa7b6804d0b323c81c9df25cd14a6124bc5d624ad3238df6875',
+       i686: '6cfa3ece1b485369f16fb62c5b17744d56a7aa162a95aab49d3d9663026cbd60',
+     x86_64: 'ac8e6a92a18862201c12563036f206804547d2a1d350f548d130bb38b9dfea5b'
+  })
+
+  depends_on 'bubblewrap'
+  depends_on 'libostree'
+  depends_on 'polkit'
+  depends_on 'appstream_glib'
+  depends_on 'libseccomp'
+  depends_on 'pyparsing'
+  depends_on 'dconf'
+
+  def self.patch
+    patch_description = 'backported fixes'
+    patch_url = 'https://patch-diff.githubusercontent.com/raw/flatpak/flatpak/pull/4132.patch'
+    patch_sha256 = '8230491be909a027fdb5a03ff0fa638331c940289fa6d68a21489d78c6f7f6b0'
+    patch_filename = 'patch'
+    puts "Downloading patch: #{patch_description}".yellow
+    system('curl', '-s', '--insecure', '-L', '-#', patch_url, '-o', patch_filename)
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless
+    Digest::SHA256.hexdigest(File.read(patch_filename)) == patch_sha256
+    puts 'patch downloaded'.lightgreen
+  end
+
+  def self.build
+    system 'env NOCONFIGURE=1 ./autogen.sh'
+    system 'filefix'
+    system "env BWRAP=#{CREW_PREFIX}/bin/bwrap CFLAGS='-flto=auto' CXXFLAGS='-flto=auto'  LDFLAGS='-flto=auto' \
+      ./configure #{CREW_OPTIONS} \
+      --with-system-install-dir=#{CREW_PREFIX}/var/lib/flatpak \
+      --enable-sandboxed-triggers \
+      --with-priv-mode=none \
+      --enable-san \
+      --without-systemd \
+      --with-system-fonts-dir=#{CREW_PREFIX}/share/fonts:/usr/share/fonts \
+      --with-system-font-cache-dirs=/usr/share/cache/fontconfig:#{CREW_PREFIX}/cache/fontconfig \
```

Comment:
Does it make sense to have a read-only cache dir?

Reply:
The read-only cache is I think generated by google for each build for built-in fonts.

Honestly my issue is actually with the  ``` --with-system-fonts-dir=#{CREW_PREFIX}/share/fonts:/usr/share/fonts \``` line, which I don't think works. I had to make a symlink in the flatpak wrapper script to generate a symlink to ~/.local/share/fonts to get fonts in ```#{CREW_PREFIX}/share/fonts``` to be seen.